src/internal/transport/websocket/WebSocketClient.ts

```diff
@@ -3,20 +3,25 @@ import axios from "axios";
 import * as promiseRetry from "promise-retry";
 import * as WebSocket from "ws";
 import { logger } from "../../util/logger";
+import { hideString } from "../../util/string";
 import { CommandIncoming, EventIncoming, isCommandIncoming, isEventIncoming } from "../RequestProcessor";
 import { sendMessage } from "./WebSocketMessageClient";
 import { RegistrationConfirmation, WebSocketRequestProcessor } from "./WebSocketRequestProcessor";
 
 export class WebSocketClient {
 
-    constructor(private registrationCallback: () => any,
-                private options: WebSocketClientOptions,
-                private requestProcessor: WebSocketRequestProcessor) {
-
-        register(this.registrationCallback, options, requestProcessor)
+    public static initialize( registrationCallback: () => any,
```

Comment:
How does this change improve the error message?

Reply:
it makes the dashboard not open after registration fails.